config/api.yml

```diff
@@ -88,6 +88,41 @@
        :post:
        - :name: create
          :identifier: alert_status_action_new
+  :alert_definition_profiles:
+    :description: Alert Definition Profiles
+    :identifier: alert_definition_profile
+    :options:
+    - :collection
+    - :subcollection
+    :verbs: *gpd
+    :klass: MiqAlertSet
+    :subcollections:
+    - :alert_definitions
+    :collection_actions:
+      :get:
+      - :name: read
+        :identifier: alert_definition_profiles_show_list
+      :post:
+      - :name: read
+        :identifier: alert_definition_profile_view
+      - :name: create
+        :identifier: alert_definition_profile_new
+      - :name: edit
+        :identifier: alert_definition_profile_edit
+      - :name: delete
+        :identifier: alert_definition_profile_delete
+    :resource_actions:
+      :get:
+      - :name: read
+        :identifier: alert_definition_profile_view
+      :post:
+      - :name: edit
+        :identifier: alert_definition_profile_edit
+      - :name: delete
+        :identifier: alert_definition_profile_delete
+      :delete:
+      - :name: delete
+        :identifier: alert_definition_profile_delete
```

Comment:
we need two kinds of assign here:
Assign to object and assign a policy to this profile.
(we cannot assign to object through an object subcollection since the target entities we need have no rest endpoints for example container_node.rb)

@abellotti me and @dkorn were discussing this and weren't sure how to do it.